patches/chrome-BUILD.gn.patch

```diff
@@ -64,21 +64,133 @@ index 960921acd7ebb0ba99092745e8d30bca76821f2a..534fe8a987d939b2dc24c8cf20d4e950
 +  # chrome_framework_version = "A"
  
    group("chrome") {
-     deps = [
-@@ -593,6 +610,12 @@ if (is_win) {
-       "--scm=1",
-       "--bundle_id=$chrome_mac_bundle_id",
-     ]
-+
+-    deps = [
+-      ":chrome_app",
+-    ]
 +    if (brave_chromium_build) {
-+      args += [
-+        "--brave_channel=" + brave_channel,
++      deps = [
++        "//brave:chrome_app",
++      ]
+ 
+-    data_deps = [
+-      ":chrome_app",
+-    ]
++      data_deps = [
++        "//brave:chrome_app",
++      ]
++    } else {
++      deps = [
++        ":chrome_app",
++      ]
++
++      data_deps = [
++        ":chrome_app",
 +      ]
 +    }
+ 
+     if (debug_devtools) {
+       deps += [ ":devtools_debug_resources" ]
+@@ -581,55 +608,59 @@ if (is_win) {
+     ]
    }
  
-   mac_app_bundle("chrome_app") {
-@@ -661,7 +684,7 @@ if (is_win) {
+-  tweak_info_plist("chrome_app_plist") {
+-    info_plist = "app/app-Info.plist"
+-    _keystone_arg = "0"
+-    if (is_chrome_branded) {
+-      _keystone_arg = "1"
++  # Disable chrome_app_plist and chrome_app targets here and they are moved to
++  # //brave to easily use other brave targets.
++  if (!brave_chromium_build) {
++    tweak_info_plist("chrome_app_plist") {
++      info_plist = "app/app-Info.plist"
++      _keystone_arg = "0"
++      if (is_chrome_branded) {
++        _keystone_arg = "1"
++      }
++      args = [
++        "--breakpad=0",
++        "--keystone=$_keystone_arg",
++        "--scm=1",
++        "--bundle_id=$chrome_mac_bundle_id",
++      ]
+     }
+-    args = [
+-      "--breakpad=0",
+-      "--keystone=$_keystone_arg",
+-      "--scm=1",
+-      "--bundle_id=$chrome_mac_bundle_id",
+-    ]
+-  }
+ 
+-  mac_app_bundle("chrome_app") {
+-    output_name = chrome_product_full_name
```

Comment:
This output name is same with `//brave:chrome_app`. So this target should be hidden.

Reply:
We definitely don't want to do this, that makes the patch even larger. Just modify `output_name` and append `-chrome` or something

Reply:
actually better yet, wrap it in an `if` with the muon build flag (don't indent)